clang/test/SemaCXX/PR41441.cpp

```diff
@@ -0,0 +1,23 @@
+// RUN: %clang --target=x86_64-pc-linux -S -fno-discard-value-names -emit-llvm -o - %s | FileCheck %s
+
+namespace std {
```

Comment:
Tests STILL haven't changed from the original patch.  Please add the regression to this test as well.